src/main/java/org/opentripplanner/routing/graphfinder/PlaceFinderTraverseVisitor.java

```diff
@@ -80,6 +81,8 @@ public PlaceFinderTraverseVisitor(
     includeVehicleRentals = shouldInclude(filterByPlaceTypes, PlaceType.VEHICLE_RENT);
     includeCarParking = shouldInclude(filterByPlaceTypes, PlaceType.CAR_PARK);
     includeBikeParking = shouldInclude(filterByPlaceTypes, PlaceType.BIKE_PARK);
+    includeStopsAndStations = shouldInclude(filterByPlaceTypes, PlaceType.STOP_OR_STATION);
```

Comment:
I think now this is true by default if no filters are defined. However, I think we maybe shouldn't have this true by default as the client software might have existing handling for when Stop type objects are returned which might break as now this sometimes would return stations which also use Stop type but have some crucial fields as null. Also again, now by default we have both STOP and STOP_OR_STATION on by default and it's a bit unclear what it means.